### DISCUSSION TITLE
Security Onion Standard Installation Problem

### DISCUSSION THREAD

**Discussion by techn00** [Security-Onion-Solutions/securityonion#7307] [Unsupported Versions] [answered]
Security Onion installation had a problem. I got the error message
![image](https://user-images.githubusercontent.com/100316438/155405611-7292a051-dab7-4685-94bf-5d09f0fffc12.png)

Found this error in sosetup.log file
so-mysql
so-mysql
2022-02-23T13:01:57Z | I | Removing so-setup permission entry from sudoers file

Upon checking so-status, Suricata is missing

![image](https://user-images.githubusercontent.com/100316438/155405746-bfff74f5-be1d-4474-9bef-b44682e9a6b7.png)

**Comment by dougburks** [comment, marked as answer]
Looks like you had another discussion for this at https://github.com/Security-Onion-Solutions/securityonion/discussions/7324 and it has been marked as answered.

In the future, please avoid creating multiple discussions for the same problem.
